src/Altinn.ApiClients.Maskinporten/Service/IMaskinporten.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace Altinn.ApiClients.Maskinporten.Services
+{
+    public interface IMaskinporten
+    {
+        /// <summary>
+        /// Generates a Maskinporten access token using a JsonWebKey
+        /// </summary>
+        Task<string> GetToken(JsonWebKey jwk, string clientId, string scope, string resource, int exp);
```

Comment:
`exp` is unused. We can only control the TTL for the assertion anyway, as the TTL for the token is controlled by scopes and client definition. The assertion TTL can probably be hardcoded to something like 10 seconds.

Reply:
@elsand i did not know that we are to able to control the token length. I though they only controlled the max. I will update code